tower-http/CHANGELOG.md

```diff
@@ -5,13 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# Unreleased
+# 0.6.0
```

Comment:
I don't think this is usually changed before release. I guess it's a good reminder that there's breaking changes, but the fact it's not out yet should also still be included. How about this?

```suggestion
# 0.6.0 (unreleased)
```